pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -628,9 +628,9 @@ message Image {
     // ID of the image.
     optional string id = 1;
     // Other names by which this image is known.
-    repeated string repo_tags = 2;
+    repeated string names = 2;
```

Comment:
> @feiskyer We can comment and require the Name to have format like image_name:tag as well. RepoTags just sounds very docker specific. Same as RepoDigests.

OK. It sounds docker specific just because only docker image is supported today. Does ACI also supports `image:tag` and `image@digest`?